spring-cloud-stream-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -492,4 +494,146 @@ Then do this:
+
+.  How do I override Spring Integration version that is coming from spring-boot-dependencies by default? +
+The following solution only works if the versions you want to override are available through a new Spring Integration BOM.
+Go to your app starter specific bom. Override the property as following:
+
+ <spring-integration.version>VERSION GOES HERE</spring-integration.version>
+
+ Then add the following in the dependencies management section in the BOM.
+
+ <dependency>
+	<groupId>org.springframework.integration</groupId>
+	<artifactId>spring-integration-bom</artifactId>
+	<version>${spring-integration.version}</version>
+	<scope>import</scope>
+	<type>pom</type>
+</dependency>
+
+
+.  How do I override spring-cloud-stream artifacts coming by default in spring-cloud-dependencies defined in core BOM? +
+The following solution only works if the versions you want to override are available through a new Spring-Cloud-Dependencies BOM.
+Go to your app starter specific bom. Override the property as following:
+
+ <spring-cloud-dependencies.version>VERSION GOES HERE</spring-cloud-dependencies.version>
+
+ Then add the following in the dependencies management section in the BOM.
+
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-dependencies</artifactId>
+        <version>${spring-cloud-dependencies.version}</version>
+        <scope>import</scope>
+        <type>pom</type>
+    </dependency>
+
+
+.  What if there is no spring-cloud-dependencies BOM available that contains my versions of spring-cloud-stream, but there is a spring-cloud-stream BOM available? +
+Go to your app starter specific BOM. Override the property as below.
+
+ <spring-cloud-stream.version>VERSION GOES HERE</spring-cloud-stream.version>
+
+ Then add the following in the dependencies management section in the BOM.
+
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-stream-dependencies</artifactId>
+        <version>${spring-cloud-stream.version}</version>
+        <scope>import</scope>
+        <type>pom</type>
+    </dependency>
+
+.  What if I want to override a single artifact that is provided through a bom? For example spring-integration-java-dsl? +
+Go to your app starter BOM and add the following property with the version you want to override:
+
+ <spring-integration-java-dsl.version>VERSION GOES HERE</spring-integration-java-dsl.version>
+
+ Then in the dependency management section add the following:
+
+    <dependency>
+        <groupId>org.springframework.integration</groupId>
+        <artifactId>spring-integration-java-dsl</artifactId>
+        <version>${spring-integration-java-dsl.version}</version>
+    </dependency>
+
+.  How do I override the boot version used in a particular app? +
+When you generate the app, override the boot version as follows.
+
+  ./mvnw clean install -PgenerateApps -DbootVersion=<boot version to override>
```

Comment:
Can you give an example showing a specific version, one would specify 1.5.2.RELEASE or 2.0.0.BUILD-SNAPSHOT ?